fixtures.csv

```diff
@@ -356,4 +356,7 @@ MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
 IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
 CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
 TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
-MATERIALS OR THE USE OR OTHER DEALINGS IN THE MATERIALS."
\ No newline at end of file
+MATERIALS OR THE USE OR OTHER DEALINGS IN THE MATERIALS."
+"  $Lua: Lua 5.1.4 Copyright (C) 1994-2008 Lua.org, PUC-Rio $","$Lua: Lua 5.1.4 Copyright (C) 1994-2008 Lua.org, PUC-Rio $"
+"$Lua: Lua 5.1.4 Copyright (C) 1994-2008 Lua.org, PUC-Rio $  ","$Lua: Lua 5.1.4 Copyright (C) 1994-2008 Lua.org, PUC-Rio $"
+"  $Lua: Lua 5.1.4  Copyright (C)   1994-2008 Lua.org, PUC-Rio $  ","$Lua: Lua 5.1.4 Copyright (C) 1994-2008 Lua.org, PUC-Rio $"
```

Comment:
Hmm there are null bytes in here, I swear! 

![image](https://user-images.githubusercontent.com/299129/82640155-f1bcdf80-9bbe-11ea-838f-492738668d4e.png)

Can you add a comment in a csv?

Reply:
Talking to yourself?

Reply:
You could make the statement something like:

`"$Lua: Lua 5.1.4 Copyright (C) 1994-2008 Lua.org, PUC-Rio, NOTE: this statement has NULL bytes $"`

It doesn't really matter what the text is.

Reply:
> Talking to yourself?

You don’t review your own code?!

Reply:
Added the note to the data.